.gitignore

```diff
@@ -1 +1,2 @@
 test.py
+LSP/
```

Comment:
`ensure_newline_at_eof_on_save` is good setting to have enabled. Even github doesn't like it:
![Screenshot 2021-06-14 at 18 26 40](https://user-images.githubusercontent.com/153197/121926287-19d52e80-cd3e-11eb-8abc-6b4cd7fde537.png)